src/components/App/App.js

```diff
@@ -2,30 +2,35 @@ import React, { Component } from 'react'
 import AllMovies from '../AllMovies/AllMovies'
 import MovieInfo from '../MovieInfo/MovieInfo'
 import './App.css'
-import movieData from '../../movieData'
 
 class App extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
-      movies: movieData.movies,
-      currentCard: null
+      movies: [],
+      currentMovie: null
     }
   }
 
   handleClick = (event) => {
-    event.preventDefault();
-    this.setState({ currentCard: Number(event.target.id) })
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${Number(event.target.id)}`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ currentMovie: data.movie })
+      })
   }
 
   returnHome = (event) => {
-    event.preventDefault();
-    this.setState({ currentCard: null })
+    this.setState({ currentMovie: null })
   }
 
-  findMovie = () => {
-    const currentMovie = this.state.movies.find(movie => movie.id === this.state.currentCard)
-    return currentMovie
+  componentDidMount() {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
```

Comment:
Same here. Be sure to move this fetch into it's own file. Additionally - These two fetches look pretty similar - can you think of any ways to DRY this up with args and params?